src/profile/components/accessibleFormikErrors/AccessibleFormikErrors.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useRef, useState } from 'react';
+import { FormikProps } from 'formik';
+import _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+
+import commonFormStyles from '../../../common/cssHelpers/form.module.css';
+import { getErrorMessageWithOptions } from '../../../common/schemas/schemas';
+import { getFormFields, FormField } from '../../helpers/formProperties';
+import { EditDataType } from '../../helpers/editData';
+
+type Props = {
+  formikProps: unknown;
+  dataType: EditDataType;
+};
+
+const AccessibleFormikErrors = (props: Props): React.ReactElement | null => {
+  const formik = props.formikProps as FormikProps<Record<string, unknown>>;
+  const [currentErrorText, setLastError] = useState<string>('');
+  const lastSubmitCountRef = useRef<number>(formik.submitCount);
+  const { t } = useTranslation();
+  const errorList = Object.keys(formik.errors);
+  const errorCount = errorList.length;
+  const formFields = getFormFields(props.dataType);
+  const errorsToText = () =>
+    errorList
+      .map(errorKey => {
+        const errorMessage = _.get(formik.errors, errorKey);
```

Comment:
formik.errors might not exist, so formik.errors[erroKey] would throw. But on the other hand in that case, the loop is not run and function is not called. So it can be formik.errors[erroKey]

Reply:
Fixed.